duckscript_sdk/src/sdk/std/process/exec/mod_test.rs

```diff
@@ -54,21 +63,12 @@ fn run_error_code_with_output() {
 }
 
 #[test]
-#[cfg(target_os = "linux")]
-fn run_get_exit_code_valid() {
-    test::run_script_and_validate(
-        vec![create("")],
-        "out = exec --get-exit-code true",
-        CommandValidation::Match("out".to_string(), "0".to_string()),
-    );
-}
+fn run_get_exit_code() {
+    let script = format!("out = exec --get-exit-code {} \"exit 42\"", COMMAND_RUNNER);
```

Comment:
before i tested valid exit and error exit. now we test only error exit.

Reply:
i just think vaild exit or error exit is the samething: they are check the exit code. so  I merged them boldly

Reply:
ya i got it. just i want to make sure i do'nt break things on different scenarios.